builder-support/dockerfiles/Dockerfile.target.ubuntu-focal

```diff
@@ -1,7 +1,15 @@
 # First do the source builds
 @INCLUDE Dockerfile.target.sdist
 
+@IF [ ${BUILDER_TARGET} = ubuntu-focal ]
```

Comment:
Is what would be built dependent on the host system?

Reply:
yes

Reply:
it was phrased as a question, but my feeling was "this might be painful". Although we have specific arch, one can use those.

Reply:
Yes, I left this one in on purpose, but perhaps we can take it out when we're done setting up multiarch in Buildbot.